tracer/test/test-applications/debugger/Samples.Probes/Samples.Probes.csproj

```diff
@@ -6,6 +6,7 @@
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <Platforms>x86;x64</Platforms>
+    <Prefer32Bit>true</Prefer32Bit>
```

Comment:
Just to make sure - did you verify that this still runs as 64 bit when we do 64 bit testing?

Reply:
This change no longer exist, resolving.